README.md

```diff
@@ -123,6 +123,22 @@ If you'd like to build the `spacetime` CLI tool from source, you can install it
 cargo install spacetimedb-cli
 ```
 
+### Installing from Source
+
+For MacOS and Linux this is straightforward. Just run the following commands in a terminal:
+
+```bash
+# Install cargo, if you already have cargo you can skip this
```

Comment:
> Might be worth noting that we explicitly require `rustup`, not just `cargo`. Users who install `cargo` via other sources (e.g. distro package managers) will still need `rustup`.
> 
> Also might be worth changing the CLI to test if the `wasm32-unknown-unknown` target is available, and only invoke `rustup` if it isn't, instead of unconditionally calling `rustup` to ensure it, so that non-`rustup` users can run SpacetimeDB as long as they install the `wasm32-unknown-unknown` target themselves.

Yeah checking for `wasm32-unknown-unknown` rather then just speculatively invoking rustup is going to have the best compatibility.